### PR TITLE
user account details

### PR DESCRIPTION
**Pull request by ocabafox**

### CODE REVIEW

**Comment by ocabafox** [comment]
@a-fis/websta-code-reviewer https://github.com/a-fis/twitter-bot-client/pull/6

File: src/app/components/Account.js [hunk 1]
```diff
@@ -1,44 +1,38 @@
...
-const Account = () =>
+const Account = ({ user }) =>
   <div className="uk-container-center uk-text-center hero-content account-background">
     <section>
       <div className="header-background">
         <div className="header-background-content">
           <img alt="sample" className="uk-border-circle profile-image" src="//scontent.cdninstagram.com/t51.2885-19/s150x150/13734387_1101995849921799_1939007949_a.jpg" />
           <div className="header-background-fields">
-            <p>OcabafoX</p>
+            <p>OcabafoX{user.email}</p>
```
Comment (jmworks) [inline src/app/components/Account.js]: this is dummy? or remaining typo?

Comment (ocabafox) [inline src/app/components/Account.js]: we don't have api endpoints yet @jmworks , just preparing the method.

Comment (jmworks) [inline src/app/components/Account.js]: ok

**Pull request merged**